packages/schemas/src/advancedText/propPanel.ts

```diff
@@ -0,0 +1 @@
+// TODO:
```

Comment:
@peteward I can't build due to this file.

```
src/advancedText/index.ts:3:27 - error TS2306: File '/Users/user/Develop/pdfme/packages/schemas/src/advancedText/propPanel.ts' is not a module.

3 import { propPanel } from './propPanel.js';
                            ~~~~~~~~~~~~~~~~

src/advancedText/propPanel.ts:1:1 - error TS1208: 'propPanel.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.

1 // TODO:



Found 2 errors in 2 files.
```

Reply:
It won't build yet, it's not finished, I was just sharing the code

Reply:
Okay